browser/about/about.md

```diff
@@ -1,4 +1,4 @@
-The Genome Aggregation Database (gnomAD), originally launched in 2014 as the Exome Aggregation Consortium (ExAC), is the result of a coalition of investigators willing to share aggregate human exome and genome sequencing data from a variety of large-scale sequencing projects, and make summary data available for the wider scientific community. The project is overseen by co-directors Heidi Rehm and Mark Daly, and steering committee members Samantha Baxter, Katherine Chao, Julia Goodrich, Konrad Karczewski, Daniel MacArthur, Benjamin Neale, Anne O'Donnell-Luria, Kaitlin Samocha, Matthew Solomonson, and Michael Talkowski. To learn more about the team that produces gnomAD please see our [team page](/team). A list of investigators and groups that have contributed data is available below.
+The Genome Aggregation Database (gnomAD™), originally launched in 2014 as the Exome Aggregation Consortium (ExAC), is the result of a coalition of investigators willing to share aggregate human exome and genome sequencing data from a variety of large-scale sequencing projects, and make summary data available for the wider scientific community. The project is overseen by co-directors Heidi Rehm and Mark Daly, and steering committee members Samantha Baxter, Katherine Chao, Julia Goodrich, Konrad Karczewski, Daniel MacArthur, Benjamin Neale, Anne O'Donnell-Luria, Kaitlin Samocha, Matthew Solomonson, and Michael Talkowski. To learn more about the team that produces gnomAD please see our [team page](/team). A list of investigators and groups that have contributed data is available below.
```

Comment:
Asking for my own information here, is there documentation about when it's necessary to use the :tm:?

Reply:
I think the main intent was to have it in a prominent location on policies, I can share the google doc from Heidi/Sam that I've been working off of -- I've pretty much just been following their edits for these content things.

Reply:
https://docs.google.com/document/d/1Stj4N9xXu5KvYeGgCuJxvvUBOjhP4eqjErryY7ZXNdc